internal/cephfs/volumeoptions.go

```diff
@@ -299,7 +300,13 @@ func newVolumeOptionsFromVolID(ctx context.Context, volID string, volOpt, secret
 	}
 
 	volOptions.ProvisionVolume = true
-	volOptions.RootPath, err = getVolumeRootPathCeph(ctx, &volOptions, cr, volumeID(vid.FsSubvolName))
+	info, err = getSubVolumeInfo(ctx, &volOptions, cr, volumeID(vid.FsSubvolName))
+	volOptions.RootPath = info.Path
```

Comment:
Processing `info.Path` in case there are errors is not useful. I would put this into an else of the err check.

Reply:
Sure, will change it.